share/smack/lib/smack.c

```diff
@@ -3,8 +3,7 @@
 
 #include <smack.h>
 #include <limits.h>
-#include <string.h>
-#include <stdlib.h>
```

Comment:
@michael-emmi Is there a particular reason that `stdlib.h` is taken away? `malloc` is used in `smack.c`.

Reply:
Fair point; this will probably lead to unnecessary warnings; adding back the `stdlib` include.